efficientdet/keras/eval.py

```diff
@@ -15,36 +15,53 @@
 # ==============================================================================
 """Eval libraries."""
 from absl import app
+from absl import flags
 from absl import logging
 import tensorflow as tf
 
 import coco_metric
 import dataloader
 import hparams_config
+import utils
 
 from keras import anchors
 from keras import efficientdet_keras
 from keras import postprocess
 
+flags.DEFINE_string('validation_file_pattern', None,
+                    'Glob for evaluation tfrecords (e.g., COCO val2017 set)')
+flags.DEFINE_string(
+    'val_json_file', None,
+    'COCO validation JSON containing golden bounding boxes.')
+flags.DEFINE_string('checkpoint', None, 'Location of the checkpoint to evaluate.')
```

Comment:
You can use yapf to format your code (max length is 80 characters)

Reply:
I tried running yapf on it and it went to 4 space indentation - is that correct?

Reply:
No, it shouldn't. You need to use style=yapf, like here: https://github.com/google/automl/blob/master/efficientdet/g3doc/faq.md#21-how-can-i-format-my-code-for-prs

Reply:
Got it!